rest-api-spec/build.gradle

```diff
@@ -28,100 +28,102 @@ testClusters.all {
+          'cat.allocation/10_basic/Column headers',
+          'cat.allocation/10_basic/Empty cluster',
+          'cat.allocation/10_basic/Node ID',
+          'cat.allocation/10_basic/One index',
+          'cat.allocation/10_basic/Select columns',
+          'cat.count/10_basic/Test cat count output',
+          'cat.fielddata/10_basic/Test cat fielddata output',
+          'cat.health/10_basic/Empty cluster',
+          'cat.health/10_basic/With ts parameter',
+          'cat.indices/10_basic/Test cat indices output',
+          'cat.indices/10_basic/Test cat indices output (no indices)',
+          'cat.indices/10_basic/Test cat indices output for closed index',
+          'cat.indices/10_basic/Test cat indices sort',
+          'cat.indices/10_basic/Test cat indices using health status',
+          'cat.indices/10_basic/Test cat indices using wildcards',
+          'cat.indices/20_hidden/Test cat indices output for dot-hidden index and dot-prefixed pattern',
+          'cat.indices/20_hidden/Test cat indices output for hidden index',
+          'cat.indices/20_hidden/Test cat indices output with a hidden index with a hidden alias',
+          'cat.indices/20_hidden/Test cat indices output with a hidden index with a visible alias',
+          'cat.indices/20_hidden/Test cat indices output with a hidden index, dot-hidden alias and dot pattern',
+          'cat.nodeattrs/10_basic/Test cat nodes attrs output',
+          'cat.nodes/10_basic/Additional disk information',
+          'cat.nodes/10_basic/Test cat nodes output',
+          'cat.nodes/10_basic/Test cat nodes output with full_id set',
+          'cat.recovery/10_basic/Test cat recovery output',
+          'cat.recovery/10_basic/Test cat recovery output for closed index',
+          'cat.repositories/10_basic/Test cat repositories output',
+          'cat.repositories/10_basic/Test cat repositories sort',
+          'cat.segments/10_basic/Test cat segments output',
+          'cat.segments/10_basic/Test cat segments using wildcards',
+          'cat.shards/10_basic/Help',
+          'cat.shards/10_basic/Test cat shards output',
+          'cat.shards/10_basic/Test cat shards sort',
+          'cat.shards/10_basic/Test cat shards using wildcards',
+          'cat.snapshots/10_basic/Help',
+          'cat.snapshots/10_basic/Test cat snapshots output',
+          'cat.tasks/10_basic/Test cat tasks output',
+          'cat.tasks/10_basic/Test cat tasks output with X-Opaque-Id',
+          'cat.templates/10_basic/Column headers',
+          'cat.templates/10_basic/Filtered templates',
+          'cat.templates/10_basic/Mixture of legacy and composable templates',
+          'cat.templates/10_basic/Normal templates',
+          'cat.templates/10_basic/Select columns',
+          'cat.thread_pool/10_basic/Test cat thread_pool output',
+          // type information about the type is removed and not passed down. The logic to check for this is also removed.
```

Comment:
Should we cleanup/remove some of these comments then given we don't plan on addressing any of these?

Reply:
I think those are beneficial. It is good to know the reason why the test is disabled without actually reading the test. The reasoning why it is disabled is clear now, but after a month or two (or earlier :D ) we might forget the details